### PR TITLE
Us1 c styling update#65

### LINKED ISSUE #65: CORRECTION: US1C - campus switching Needs proper styling
- [x] Campus switch should be pill shaped
- [x] Semi-transparent (opacity/fill)
- [x] Current campus should be highlighted

### PR DESCRIPTION
**Pull request by GD-Dheer**
All required corrections are complete: 
 - Campus switch should be pill shaped
 - Semi-transparent (opacity/fill)
 - Current campus should be highlighted

UI matches the following: 
image

### CODE REVIEW

**Review by eufekt — CHANGES_REQUESTED** [review]
I am getting weird rendering both on android and ios. The buttons appear on top of the search bar. Please fix
IMG_4424 2

**Comment by trevornag** [comment]
Just tried it. I am getting the same issue as @Nichita-Hariton. I glanced at the code and for the most for the CSS, I see percentages but try as much as possible to use percentages rather than concrete values i.e. 44. This has caused styling issues in the past whereby it looks good on one device but completely off on another. Perhaps changing properties to % will solve this problem

File: components/switchCampuses/styles.js [hunk 1]
```diff
@@ -1,16 +1,82 @@
...
+    borderBottomLeftRadius: 20,
+    borderBottomRightRadius: 0,
+    borderTopLeftRadius: 20,
+    borderTopRightRadius: 0,
+    margin: 1,
+    opacity: 0.75,
+    backgroundColor: '#ffffff',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
```
Comment (trevornag) [inline components/switchCampuses/styles.js]: maybe replace concrete values with percentages

**Review by trevornag — CHANGES_REQUESTED** [review]

**Comment by GD-Dheer** [comment]
Updated. Verified and working on different Iphone

**Review by eufekt — APPROVED** [review]
COde looks good, please run  ```$yarn lint:fix``` to get rid of spaces

**Review by mohhef — APPROVED** [review]
Required corrections are made. It seems to work fine.

**Review by gisellemartel — CHANGES_REQUESTED** [review]
Tested on a iPhone XR seems to be issues with the UI. Please refer to screenshot: 
IMG_1250

**Review by eufekt — APPROVED** [review]
Please fix iphone xr latest issue. Approved.

**Review by trevornag — APPROVED** [review]

**Review by gisellemartel — APPROVED** [review]

**Pull request merged**